### ISSUE TITLE
Close this project?

### ISSUE THREAD

**Issue by yongli-abc** [xukai92/Teach-Each-Other#1]
I suggest close this project for now.
It didn't turn out very well!
Sigh!

**Comment by xukai92** [comment]
Hehe. wanna restart?

**Comment by yongli-abc** [comment]
Not really, it seems our knowledge stacks are diverging further and further, frankly speaking it would be impossible to learn other's knowledge merely by personal tutoring...

**Comment by yongli-abc** [comment]
I'm trying to be more active on github and participate in a few open source projects in the future. Since I've finished the hell CFA exam, I should be able to concentrate more on tech side.

If you have project ideas, I can participate in a more OS-style, like helping you fix issues, etc...
